iphone/Maps/Core/iCloud/CloudStorageManager.swift

```diff
@@ -271,21 +270,21 @@ private extension CloudStorageManager {
            .fileNotUploadedDueToQuota,
            .ubiquityServerNotAvailable:
         LOG(.warning, "Synchronization Warning: \(syncError.localizedDescription)")
-        synchronizationError = syncError
+        self.synchronizationError = syncError
       case .iCloudIsNotAvailable:
         LOG(.warning, "Synchronization Warning: \(error.localizedDescription)")
-        stopSynchronization()
+        self.stopSynchronization()
       case .failedToOpenLocalDirectoryFileDescriptor,
            .failedToRetrieveLocalDirectoryContent,
            .containerNotFound,
            .failedToCreateMetadataItem,
            .failedToRetrieveMetadataQueryContent:
         LOG(.error, "Synchronization Error: \(error.localizedDescription)")
-        stopSynchronization(withError: error)
+        self.stopSynchronization(withError: error)
       }
     default:
-      LOG(.debug, "Non-synchronization Error: \(error.localizedDescription)")
-      stopSynchronization(withError: error)
+      LOG(.error, "Non-synchronization Error: \(error.localizedDescription)")
```

Comment:
There are no other cases in this switch, right? Should this default be removed to detect when a new case is added instead of using a default here?

Reply:
Default errors are all the system error (not the `SynchronizationError` instances - this is why there is only the one case). 
There can be a bunch of different system errors.

Reply:
Maybe then write "System error: \(error.localizedDescription)" ?